src/gui/previewlistdelegate.cpp

```diff
@@ -61,30 +54,19 @@ void PreviewListDelegate::paint(QPainter *painter, const QStyleOptionViewItem &o
         break;
 
     case PreviewSelectDialog::PROGRESS:
-    {
+        {
             const qreal progress = (index.data().toReal() * 100);
+            const QString text = (progress >= 100)
```

Comment:
Seems creating text representation of progress should be done in ProgressBarPainter. Accepting both number and appropriate text makes it to allow inconsistent usage.

Reply:
>Seems creating text representation of progress should be done in ProgressBarPainter. 

That would call for more changes due to https://github.com/qbittorrent/qBittorrent/pull/14543/files#diff-b4880d11d27223fcf763037ec300bd23c6a274429e975e3bb0906369aa5d08e1R71-R73
I don't want to handle it in this PR.